### PR TITLE
feat(hz): Add "pb_json_tag" to make the generated json tag naming style consistent with "protoc-gen-go"

### PR DESCRIPTION
**Pull request by FGYFFFF**
#### What type of PR is this?

feat
#### Check the PR title.

- [ ] This PR title match the format: \<type\>(optional scope): \<description\>
- [ ] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) Translate the PR title into Chinese.
添加 "pb_json_tag" 来使生成的 json tag 命名方式对齐 protoc-gen-go

#### (Optional) More detail description for this PR(en: English/zh: Chinese).

en: Add "pb_json_tag" to make the generated json tag naming style consistent with "protoc-gen-go"
zh(optional): 添加 "pb_json_tag" 来使生成的 json tag 命名方式对齐 protoc-gen-go

### CODE REVIEW

File: cmd/hz/internal/protobuf/tags.go [hunk 1]
```diff
@@ -51,6 +52,9 @@ func CheckTagOption(args *config.Argument) (ret []generator.Option) {
 	if args.JSONEnumStr {
 		ret = append(ret, generator.OptionMarshalEnumToText)
 	}
+	if args.PbJsonTag {
```
Comment (welkeyever) [inline cmd/hz/internal/protobuf/tags.go:55]: Json -> JSON? naming consistency

Comment (FGYFFFF) [reply cmd/hz/internal/protobuf/tags.go:55]: Most of the code about "json" is named "json" or "Json". Here we replace "JSON" with "Json"

File: cmd/hz/internal/protobuf/tags.go [hunk 2]
```diff
@@ -258,7 +262,11 @@ func m2s(mt model.Tag) (ret [2]string) {
 
 func reflectJsonTag(f protoreflect.FieldDescriptor) (ret model.Tag) {
 	ret.Key = "json"
-	ret.Value = checkSnakeName(f.Name())
+	if pbJsonTagStyle {
+		ret.Value = checkSnakeName(string(f.Name()))
+	} else {
+		ret.Value = checkSnakeName(f.JSONName())
```
Comment (FGYFFFF) [inline cmd/hz/internal/protobuf/tags.go:268]: Here "JSON" is other library!

File: cmd/hz/internal/app/app.go [hunk 3]
```diff
@@ -117,8 +117,9 @@ func Init() *cli.App {
 	protoPluginsFlag := cli.StringSliceFlag{Name: "protoc-plugins", Usage: "Specify plugins for the protoc. ({plugin_name}:{options}:{out_dir})"}
 	noRecurseFlag := cli.BoolFlag{Name: "no_recurse", Usage: "Generate master model only.", Destination: &globalArgs.NoRecurse}
 
-	jsonEnumStrFlag := cli.BoolFlag{Name: "json_enumstr", Usage: "Use string instead of num for json enums when idl is thrift.", Destination: &globalArgs.JSONEnumStr}
+	jsonEnumStrFlag := cli.BoolFlag{Name: "json_enumstr", Usage: "Use string instead of num for json enums when idl is thrift.", Destination: &globalArgs.JsonEnumStr}
 	unsetOmitemptyFlag := cli.BoolFlag{Name: "unset_omitempty", Usage: "Remove 'omitempty' tag for generated struct.", Destination: &globalArgs.UnsetOmitempty}
+	pbJsonTag := cli.BoolFlag{Name: "pb_json_tag", Usage: "Name style for json tag alignments protoc-gen-go(Only works protobuf).", Destination: &globalArgs.PbJsonTag}
```
Comment (welkeyever) [inline cmd/hz/internal/app/app.go:122]: 这个行为为啥不默认为true呢

Comment (FGYFFFF) [reply cmd/hz/internal/app/app.go:122]: 兼容之前的行为；不过确实默认值为 true 会合理些。我改下

Comment (FGYFFFF) [reply cmd/hz/internal/app/app.go]: 命令行这里尽量不用反向逻辑， 选项改成 pb_camel_json_tag

File: cmd/hz/internal/app/app.go [hunk 4]
```diff
@@ -117,8 +117,9 @@ func Init() *cli.App {
 	protoPluginsFlag := cli.StringSliceFlag{Name: "protoc-plugins", Usage: "Specify plugins for the protoc. ({plugin_name}:{options}:{out_dir})"}
 	noRecurseFlag := cli.BoolFlag{Name: "no_recurse", Usage: "Generate master model only.", Destination: &globalArgs.NoRecurse}
 
-	jsonEnumStrFlag := cli.BoolFlag{Name: "json_enumstr", Usage: "Use string instead of num for json enums when idl is thrift.", Destination: &globalArgs.JSONEnumStr}
+	jsonEnumStrFlag := cli.BoolFlag{Name: "json_enumstr", Usage: "Use string instead of num for json enums when idl is thrift.", Destination: &globalArgs.JsonEnumStr}
```
Comment (welkeyever) [inline cmd/hz/internal/app/app.go:120]: 这个别改了？

Comment (FGYFFFF) [reply cmd/hz/internal/app/app.go]: done

File: cmd/hz/internal/config/argument.go [hunk 5]
```diff
@@ -51,8 +51,9 @@ type Argument struct {
 	Gopkg       string // $GOPATH/src/{{gopkg}}
 	ServiceName string // service name
 
-	JSONEnumStr     bool
+	JsonEnumStr     bool
```
[the remark this answers is not in the document]
Comment (FGYFFFF) [reply cmd/hz/internal/config/argument.go]: DONE

**Review by Duslia — APPROVED** [review]

**Review by welkeyever — APPROVED** [review]

**Pull request merged**